flow/connectors/mysql/qvalue_convert.go

```diff
@@ -366,7 +425,22 @@ func QValueFromMysqlRowEvent(mytype byte, qkind qvalue.QValueKind, val any) (qva
 		case qvalue.QValueKindJSON:
 			return qvalue.QValueJSON{Val: val}, nil
 		case qvalue.QValueKindGeometry:
-			// TODO figure out mysql geo encoding
+			// For string representation of geometry
```

Comment:
We can probably remove this code path, I think string is only really passed down from go-mysql for time types